engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -545,4 +545,6 @@ public SolApplication getSolApplication() {
     public HullConfigManager getHullConfigManager() {
         return hullConfigManager;
     }
+
+    public boolean isInvincible(){return hero.isInvincible();}
```

Comment:
First - formatting
Second - why is this method even here? Heroes invincibility is not a property of the SolGame